data/os/Fedora.yaml

```diff
@@ -2,3 +2,4 @@
 letsencrypt::configure_epel: false
 letsencrypt::plugin::dns_rfc2136::package_name: 'python3-certbot-dns-rfc2136'
 letsencrypt::plugin::dns_route53::package_name: 'python3-certbot-dns-route53'
+letsencrypt::plugin::dns_cloudflare::package_name: 'python3-certbot-dns-cloudflare'
```

Comment:
This is partially pre-existing, but I think RedHat-family.yaml should cover all of this.

Reply:
Will remove it 👍